src/stat.cc

```diff
@@ -764,8 +769,9 @@ void
 DumpMallocStatistics(StoreEntry* sentry)
 {
 #if XMALLOC_STATISTICS
-    xm_deltat = current_dtime - xm_time;
-    xm_time = current_dtime;
+    using namespace std::chrono;
+    xm_deltat = system_clock::now() - xm_time;
+    xm_time = system_clock::now();
```

Comment:
Squid is using "cached current time" by default (i.e. current_time and friends). This old code was using cached current time. Until a different approach is proposed and agreed on, please continue to use cached current time when upgrading to std::chrono API instead of switching to uncached now().